.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: File a bug report
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please fill the form below.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: reproducible
+    attributes:
+      label: How can we reproduce this?
+      description: |
+        Please share a public repository that reproduces the issue.
```

Comment:
Maybe make it sound a little less restrictive: "share a code snippet, gist, or public repo" - depending on the size of the problem a small snippet within the ticket is fine with me.

Reply:
ah, good one